src/python/grpcio_tests/tests_aio/unit/server_test.py

```diff
@@ -506,6 +506,14 @@ async def test_maximum_concurrent_rpcs(self):
         await channel.close()
         await server.stop(0)
 
+    async def test_is_active(self):
+        if (self._server.context.is_active()):
+            pass
```

Comment:
We need stronger test cases:

- Check if `is_active()` returns False, when RPC succeeds;
- Check if `is_active()` returns False, when RPC fails.

You can use callbacks to implement them.